spec.md

```diff
@@ -2232,6 +2331,58 @@ message VolumeUsage {
   // Units by which values are measured. This field is REQUIRED.
   Unit unit = 4;
 }
+
+message VolumeHealth {
+  option (alpha_message) = true;
+
+  enum Condition {
+    // Volume health condition is unknown; If this is the initial
+    // report of volume health, treat it as healthy. If there was
+    // already a condition reported previously, treat unknown as
+    // unchanged from the previous condition.
+    UNKNOWN = 0;
+    // Volume is accessible and no problems are detected. 
+    HEALTHY = 1;
+    // Volume is not accessible currently, but there's no reason to
+    // expect data loss. Examples: power outage, network outage,
+    // maintenance.
+    TEMPORARY_INACCESSIBLE = 2;
+    // Volume is accessible currently, but is degraded. Data loss
+    // is unlikely as the storage system is rebuilding to fix the
```

Comment:
s/is rebuilding/may be rebuilding/

it actually may not be automatic, and operator intervention may be required here - right? if not, and the intent is to differentiate between states that automation will recover vs. those that a human will recover then we probably need another state here.

i'm leaning towards NOT adding a new state to differentiate, and simply update the wording here instead.

Reply:
Yes, it is possible that operator intervention may be needed.  I changed to "may be rebuilding".